src/components/HealthClimateComponents/BlackCarbon.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { Grid, Typography } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = theme => ({
+  grid: {
+    height: '45rem'
```

Comment:
Again, if setting `height`, ensure it will work in a mobile view. Otherwise, let content dictate the `height`.

Reply:
@kilemensi I set the height to coordinate all three column content. Without the height, the content alignment is off, especially at the top of the grid column. I'll have to add breakpoints to fix this.